cocoapods_installer_test.go

```diff
@@ -0,0 +1,150 @@
+package main
+
+import (
+	"strings"
+	"testing"
+
+	"bitrise-steplib/steps-cocoapods-install/mocks"
+
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+)
+
+func Test_GivenCocoapodsInstaller_WhenArgsGiven_ThenRunsExpectedCommand(t *testing.T) {
```

Comment:
This function tests the fist line of the `InstallPods` method:
```
func (i CocoapodsInstaller) InstallPods(podArg []string, podCmd string, podfileDir string, verbose bool) error {
    if err := i.runPodInstall(podArg, podCmd, podfileDir, verbose); err == nil {
        return nil
    } else {
        ...
    }
   ...
}
```
It feels like it would be great to have one where the fist install command fails and then a repo update plus again an install command happens. Just to be sure we are using the right commands all the way.

Reply:
New test case added in: https://github.com/bitrise-steplib/steps-cocoapods-install/pull/90/commits/89439d51f4d68674c8afbad780c2154023dbb110